content/developer/reference/standard_modules/account.rst

```diff
@@ -1,4 +1,4 @@
-:nosearch:
+.. _reference/account:
```

Comment:
You should ideally use the `:doc:` role rather than `:ref:` to reference entire pages. This prevents undesired auto scrolls after redirection.